roles/common/tasks/epel.yml

```diff
@@ -16,6 +16,12 @@
   register: epel_repo
   with_dict: epel_repos
 
+- name: Configure local epel mirrorlists
+  template: src={{ item }} dest=/etc/yum.repos.d/{{ item }} owner=root group=root mode=0644
```

Comment:
nitpick: I prefer this kind of formatting:

```
template:
  src: "{{ item }}"
  dest: "/etc/yum.repos.d/{{ item }}"
...
```

I think it's easier to read and makes for better diffs if/when we ever change this code.